application/src/main/java/uk/gov/hmcts/dm/hateos/DocumentContentVersionHalResource.java

```diff
@@ -45,7 +45,7 @@ public DocumentContentVersionHalResource(DocumentContentVersion documentContentV
 
         add(linkTo(methodOn(DocumentContentVersionController.class).getDocumentContentVersionDocumentBinary(
                 documentContentVersion.getStoredDocument().getId(),
-                documentContentVersion.getId())).withRel("binary"));
+                documentContentVersion.getId(),null)).withRel("binary"));
 
```

Comment:
I think you may need a space after comma or some tools may complain

Reply:
thanks